src/adapter.rs

```diff
@@ -1235,11 +1451,106 @@ mod tests {
 
     #[test]
     fn rustdoc_json_format_version() {
-        let current_crate = load_rustdoc_from_file(Path::new("./test_data/rustdoc_v21.json"));
+        let version = rustdoc_types::FORMAT_VERSION;
+        let current_crate =
+            load_rustdoc_from_file(Path::new(&format!("./test_data/rustdoc_v{version}.json")));
```

Comment:
Won't this change cause a worse error than the original? With this change it seems like we'd get a "file not found" error when there's a version mismatch, instead of a "version mismatch" error.

Reply:
I'm sorry for this, the change is completely unintentional... At some moment I accidentally removed this test, but noticed it only later and wasn't able to simply undo, so I copied it... but from the wrong branch.